src/leakbox.rs

```diff
@@ -33,6 +34,10 @@ impl<T> LeakBox<T> {
         self.ptr.cast()
     }
 
+    pub fn non_null(&self) -> NonNull<T> {
```

Comment:
nitpicky, TIOLI: should this maybe be `as_non_null` per the [naming guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv)?